float32[] joint_positions

